kie-server-parent/kie-server-services/kie-server-services-jbpm-ui/src/main/resources/form-templates-providers/js/kieserver-ui.js

```diff
@@ -52,18 +51,22 @@ function startProcess(button) {
 		    if(typeof parent.afterProcessStarted === "function") {
                 parent.afterProcessStarted(msg);
             }
-		    
+
 		}).fail(function( xhr, msg, error ) {
 			showFailure('Something went wrong ' + error + ' ( ' + xhr.responseText + ')');
 			button.disabled = false;
 		});
 	}
 }
 
+function getCorrelationKey() {
+	return window.parent.document.getElementById("kie-server-from-correlationKey").value;
```

Comment:
@bxf12315 Good point here! I forgot the form is self-contained and the correlation key belongs to the opener app.

I personally don't like this approach because we are expecting the parent to have an input with a specific id... this could end up with issues if a customer is embedding this form in his own app. Instead of that, why not follow the pattern to perform an action after starting the process:
https://github.com/kiegroup/droolsjbpm-integration/pull/2722/files#diff-241bb47e8c6efc7061a74acce3276e22ee630db29c0e7c5beb7f3b7af5536781R51
we may check if the parent has a function called getCorrelationKey and if exists get the value from there.